dll/win32/shell32/dialogs/filedefext.cpp

```diff
@@ -1353,19 +1415,26 @@ CFileDefExt::CountFolderAndFiles(HWND hwndDlg, LPWSTR pwszBuf, UINT cchBufMax, D
             FileSize.u.LowPart  = wfd.nFileSizeLow;
             FileSize.u.HighPart = wfd.nFileSizeHigh;
             m_DirSize.QuadPart += FileSize.QuadPart;
+            // Calculate size on disc
+            if (!GetPhysicalFileSize(sFileName.GetBuffer(), &FileSize))
+                ERR("GetPhysicalFileSize failed for %ls\n", sFileName.GetBuffer());
```

Comment:
`GetString()` is required here, it cannot cast automatically here

Reply:
ok